File: frontEnd/src/app/types/employer.ts
export class Employer {
  public EmployerID: number;
  public Name: string;
  public Address: string;
  public City: string;
  public Postcode: string;
  public Email: string;
  public Password: string;

  constructor(
    applicationID: number,
    name: string,
    address: string,
    city: string,
    postcode: string,
    email: string,
    password: string,
  ) {
    this.EmployerID = applicationID;
    this.Name = name;
    this.Address = address;
    this.City = city;
    this.Postcode = postcode;
    this.Email = email;
    this.Password = password;
  }
}

File: backend/BackendService/Model/Employer.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace BackendService.Model
{
    [Table("employer")]
    public class Employer
    {
        [Key]
        public int? EmployerID { get; set; }
        public string Name { get; set; }
        public string Address { get; set; }
        public string City { get; set; }
        public string Postcode { get; set; }
        public string Email { get; set; }
        public string Telephone { get; set; }
    }
}
